### PR TITLE
Hide Stripe onboarding if no public repos

### LINKED ISSUE #458: Stripe Connect: Only show for organizations with open source repositories
No need otherwise

### PR DESCRIPTION
**Pull request by hult**
Fixes #458

### CODE REVIEW

File: clients/apps/web/src/components/Dashboard/Account/Topbar.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { Visibility } from '@/../../../packages/polarkit/src/api/client/models/Visibility'
```
Comment (zegl) [inline clients/apps/web/src/components/Dashboard/Account/Topbar.tsx:1]: What happened here? 😂 Crazy import path!

Comment (hult) [reply clients/apps/web/src/components/Dashboard/Account/Topbar.tsx:1]: Ouch. Auto-generated by vscode. Will fix.

Comment (hult) [reply clients/apps/web/src/components/Dashboard/Account/Topbar.tsx:1]: Bah. 
image

Comment (hult) [reply clients/apps/web/src/components/Dashboard/Account/Topbar.tsx:1]: Fixed in #527

**Pull request merged**